tests/diff-tests/StackAr/Makefile

```
@@ -9,14 +9,14 @@ test:
-	cp -f StackAr-2a.inv.gz StackAr-2.inv.gz 
+	cp -f StackAr-1a.inv.gz StackAr-1.inv.gz
+	cp -f StackAr-2a.inv.gz StackAr-2.inv.gz
 
 
 bad:
-	cp -f StackAr-1b.inv.gz StackAr-1.inv.gz 
-	cp -f StackAr-2b.inv.gz StackAr-2.inv.gz 
+	cp -f StackAr-1b.inv.gz StackAr-1.inv.gz
+	cp -f StackAr-2b.inv.gz StackAr-2.inv.gz
 
 worse:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)